halconfig/front50.yml

```diff
@@ -0,0 +1,19 @@
+server:
+  port: ${services.front50.port:8080}
+  address: ${services.front50.host:localhost}
+
+hystrix:
+  command:
+    default.execution.isolation.thread.timeoutInMilliseconds: 15000
+  threadpool:
+    DefaultNotificationDAO:
+      coreSize: 25
+      maxQueueSize: 100
+    DefaultPipelineDAO:
+      coreSize: 25
+      maxQueueSize: 100
+
+cassandra:
```

Comment:
With our deprecation of cassandra is it OK to leave this out? @skim1420 @duftler